paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -513,7 +513,17 @@ internal class CustomerSheetViewModel @Inject constructor(
         viewModelScope.launch {
             createPaymentMethod(paymentMethodCreateParams)
                 .onSuccess { paymentMethod ->
-                    attachPaymentMethodToCustomer(paymentMethod)
+                    if (paymentMethod.type == PaymentMethod.Type.USBankAccount &&
```

Comment:
Can we pull this into an extension property to provide more context?

```suggestion
                    if (paymentMethod.isUnverifiedUSBankAccount) {
```